pimd/pim_iface.c

```diff
@@ -54,9 +54,9 @@
 
 #if PIM_IPV == 4
 static void pim_if_igmp_join_del_all(struct interface *ifp);
-static int igmp_join_sock(const char *ifname, ifindex_t ifindex,
-			  struct in_addr group_addr, struct in_addr source_addr,
-			  struct pim_interface *pim_ifp);
+static int gm_join_sock(const char *ifname, ifindex_t ifindex,
+			struct in_addr group_addr, struct in_addr source_addr,
```

Comment:
Need to change struct in_addr to pim_addr and also please remove gm_join_sock api out of  f PIM_IPV == 4